span_sink.go

```diff
@@ -23,6 +23,7 @@ const datadogResourceKey = "resource"
 const datadogNameKey = "name"
 
 const lightStepOperationKey = "name"
+const lightstepMultiplexTracerNum = 10
```

Comment:
We should make this config so we can dial in different hosts to have appropriately sized lists!

Reply:
Merging now so we can start rolling across QA, and created a ticket to add a config option